assets/scss/partials/base.scss

```diff
@@ -1,7 +1,6 @@
 html {
     font-size: 62.5%;
     overflow-y: scroll;
-    scroll-behavior: smooth;
```

Comment:
> because Chromium does not support smooth scrolling on both the main page via an a link and on another element via a scrollTo() call.

I re-added this line and removed `smoothAnchors.ts`, tried on Chromium 99.0.4839.0 on Windows 11, and don't see the issue (smooth scroll still working if I click on TOC link). Can you give me some information about this?

Reply:
Will recheck, it might have been patched

Reply:
Okay, seems like this is still the issue. To reproduce, you can take a footnote link in a long document, the animation will glitch when clicking on the footnote link and "snap" to the next section because the animation in the ToC stops the ongoing scroll animation in the main document.

Here's a video illustrating this:

https://user-images.githubusercontent.com/5175705/150031692-42cf2234-392f-4611-a960-3eefdea9e97b.mp4

Reply:
Here are some links related to this issue. I will also add them in the `smoothAnchors` file to properly document the reason for that file's existence.

- https://stackoverflow.com/questions/49318497/google-chrome-simultaneously-smooth-scrollintoview-with-more-elements-doesn
- https://stackoverflow.com/questions/57214373/scrollintoview-using-smooth-function-on-multiple-elements-in-chrome
- https://bugs.chromium.org/p/chromium/issues/detail?id=833617
- https://bugs.chromium.org/p/chromium/issues/detail?id=1043933
- https://bugs.chromium.org/p/chromium/issues/detail?id=1121151

Reply:
I see, thanks for the information!